.vscode/settings.json

```diff
@@ -1,7 +1,14 @@
 {
   "tailwindCSS.experimental.configFile": {
-    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
-    "apps/storybook-react-native/tailwind-intellisense.config.js": "packages/design-system-react-native/src/**"
+    "apps/storybook-react/tailwind.config.js": [
+      "apps/storybook-react/**",
+      "packages/design-system-react/src/**",
```

Comment:
Tailwind intellisense and prettier work as expected in `design-system-react`

https://github.com/user-attachments/assets/22e64d76-f175-42b3-9c34-8995413bc886